src/content/post/2025-05-03-fix-create-github-app-token.md

```
@@ -0,0 +1,51 @@
+title: "create-github-app-tokenのバグをなおした"
+date: 2025-05-03
+topics: ["github actions"]
+excerpt: ''
+type: tech
+---
+
+[actions/create-github-app-token](https://github.com/actions/create-github-app-token)が2.0.4になったタイミングでCIが落ちるようになってしまった。
+
+なんやかんやで直したので以下に備忘録を書く。
+
+https://github.com/Omochice/nur-packages/issues/86
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)